infrastructure/state.tf

```diff
@@ -8,7 +8,7 @@ terraform {
     }
     azuread = {
       source  = "hashicorp/azuread"
-      version = "1.6.0"
+      version = ">= 2.0.0"
```

Comment:
after this did you try in demo ?

Reply:
Yes, tested in demo [build](https://build.platform.hmcts.net/view/BSP/job/HMCTS_BSP/job/bulk-scan-processor/job/demo/534/console)